noreact.d.ts

```diff
@@ -0,0 +1,2 @@
+export { default as default } from "./types/index";
```

Comment:
Shouldn't this be in the `/types` folder not the root?
Or does it need to be here for the exports, I'm assuming it's for something like `import ga from 'react-ga/noreact`;`